src/lib/runtime_config/runtime_config.ml

```diff
@@ -1359,14 +1359,16 @@ let make_fork_config ~staged_ledger ~global_slot ~state_hash ~blockchain_length
       }
     ~proof:(Proof_keys.make ~fork ()) ()
 
-let slot_tx_end_or_default, slot_chain_end_or_default =
-  let f compile get_runtime t =
-    Option.map ~f:Mina_numbers.Global_slot.of_int
-    @@ Option.value_map t.daemon ~default:compile ~f:(fun daemon ->
-           Option.merge compile ~f:(fun _c r -> r) @@ get_runtime daemon )
-  in
-  ( f Mina_compile_config.slot_tx_end (fun d -> d.slot_tx_end)
-  , f Mina_compile_config.slot_chain_end (fun d -> d.slot_chain_end) )
+let slot_tx_end_or_default _ = None
```

Comment:
Is there a guarantee that the associated logic works correctly with 'None'? I'll bet that simply increasing the default constant wherever it is defined would make for a smaller diff, and more likely to work with fewer changes.

Reply:
I went to change the config values but `slot_tx_end` and `slot_chain_end` are undefined there